Parsely/wp-parsely
# Add Pull Request Template & Issue Templates for Bug Reports & Feature Requests

## Description

This adds a [Pull Request Template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) so we can add some bare bones structure to the formatting of change proposals.

See also: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates#pull-request-templates

It also adds templates for Bug Reports & Feature Requests.

This were sourced from [@GaryJones' plugin boilerplate](https://github.com/GaryJones/plugin-boilerplate/tree/353e4fa4071e163a42f7b233983dc25dc50d2076/.github).

## To Test

* Make sure the template makes sense and the file name format is followed correctly.
* Propose any changes you'd like to see.
